tests/features/generated_story_tables.feature

```diff
@@ -0,0 +1,65 @@
+@story_tables
+Feature: Generated story table
+
+To cover:
+- [x] Trigger action
+- [x] Basic direct fields
+- [ ] Fields created with objects
+- [ ] Fields triggered with index vars
+- [ ] Fields triggered with nested index vars
+- [ ] Fields triggered with generic object vars
+- [x] Able to select first choice (not thorough)
+- [x] Able to select second choice (not thorough)
+- [x] Able to select third choice (not thorough)
+- [ ] Able to select none of the above
+- [x] Hidden fields
+- [x] Nested hidden fields listed out of order
+- [x] Continue button field listed before unrequired fields
+
+TODO:
+- Check that all the appropriate items are selected somehow. Maybe we need to get actual screen vars into a report. That's a lot of noise, though.
+
+The below scenario covers:
+- [x] Trigger action
+- [x] Basic direct fields
+- [x] Able to select first choice (not thorough)
+- [x] Able to select second choice (not thorough)
+- [x] Able to select third choice (not thorough)
+- [x] Hidden fields
+- [x] Nested hidden fields listed out of order
+- [x] Continue button field listed before unrequired fields
+
+@generated @slow @1 @out_of_order
+Scenario: Fields are listed out of order
```

Comment:
Specify that "Fields are listed out of order **so this test will generate an error**" or something similar?

Reply:
Fields out of order should not generate an error. We were just testing that this particular occasion. Our code (not the test) caused an error incorrectly, which then triggered this infinite loop. This was just a happy circumstance that let us fix the problem with the infinite loop. That's why I kept the error around instead of making a PR with them both fixed - we could see this behaving properly. In the future we'll have to work out testing our framework's errors (which is different than the errors that developers should be testing).

Does that make sense? I'm not sure I articulated it very well. We can discuss and clarify more if needed and add it in here posthumously.

Reply:
Yeah it makes sense now :) This is one of the testing-framework-tests we should write then at some point in the future.